mlflow/models/utils.py

```diff
@@ -66,6 +66,13 @@
 EXAMPLE_DATA_KEY = "inputs"
 EXAMPLE_PARAMS_KEY = "params"
 EXAMPLE_FILENAME = "input_example.json"
+SERVING_INPUT_PATH = "serving_input_path"
```

Comment:
Why do we need to save a separate file?

Reply:
Because this comment https://github.com/mlflow/mlflow/pull/12625#discussion_r1671919732
I think we should have `input_example` file to store an example that can be directly feed into `model.predict`, and `serving_input` file to store a json object that can be directly passed to serving.

Reply:
Thank you for the explanation, so we save the latter so it can be directly copy-and-paste to the serving UI / API request. Can we make the file name to be self-explanatory? The term "serving" is contextual, but it needs to be clear to users as well (otherwise we may get questions like those for `metadata` folder). I'd rename it to `scoring_server_input_payload.json` (or anything you think is better!)

Reply:
I think people are more familiar with the word `serving` than `scoring_server` since databricks just use the name `model serving` 🤔 Does `serving_input_payload.json` sound better?

Reply:
Hmm that's true... yea `serving_input_payload.json` sounds fine🙂

Later we can probably rename the old one to `predict_input_example.json` after updating the serving UI to make the difference between two clearer.